### PR TITLE
Task-37622: Implement Dlp feature V0

### PR DESCRIPTION
**Pull request by azayati**

### CODE REVIEW

File: core/core-configuration/src/main/webapp/WEB-INF/conf/wcm-core/core-connector-configuration.xml [hunk 1]
```diff
@@ -132,5 +132,60 @@
...
+      <init-params>
+        <object-param>
+          <name>actions</name>
+          <object type="org.exoplatform.services.jcr.impl.ext.action.AddActionsPlugin$ActionsConfig">
+            <field name="actions">
+              <collection type="java.util.ArrayList">
+                <value>
+                  <object type="org.exoplatform.services.jcr.impl.ext.action.ActionConfiguration">
+                    <field name="workspace"><string>collaboration</string></field>
+                    <field name="eventTypes"><string>addNode,addProperty,changeProperty,removeProperty</string></field>
```
Comment (rdenarie) [inline core/core-configuration/src/main/webapp/WEB-INF/conf/wcm-core/core-connector-configuration.xml]: Why we should launch the action on removeProperty event ?

File: core/core-configuration/src/main/webapp/WEB-INF/conf/wcm-core/core-connector-configuration.xml [hunk 2]
```diff
@@ -132,5 +132,60 @@
...
+    <target-component>org.exoplatform.commons.dlp.processor.DlpOperationProcessor</target-component>
+    <component-plugin>
+      <name>FileDlpConnector</name>
+      <set-method>addConnector</set-method>
+      <type>org.exoplatform.ecm.connector.dlp.FileDlpConnector</type>
+      <description>File Dlp Connector</description>
+      <init-params>
+        <value-param>
+          <name>dlp.keywords</name>
+          <value>${exo.dlp.keywords:}</value>
```
Comment (rdenarie) [inline core/core-configuration/src/main/webapp/WEB-INF/conf/wcm-core/core-connector-configuration.xml:179]:
```suggestion
          <value>${exo.dlp.keywords}</value>

```

**Review by rdenarie — CHANGES_REQUESTED** [review]

File: core/core-configuration/src/main/webapp/WEB-INF/conf/wcm-core/core-connector-configuration.xml (hunk 2 above)
Comment (azayati) [reply core/core-configuration/src/main/webapp/WEB-INF/conf/wcm-core/core-connector-configuration.xml:179]: not working with ${exo.dlp.keywords}

File: core/core-configuration/src/main/webapp/WEB-INF/conf/wcm-core/core-connector-configuration.xml (hunk 1 above)
Comment (azayati) [reply core/core-configuration/src/main/webapp/WEB-INF/conf/wcm-core/core-connector-configuration.xml]: Removed

**Review by rdenarie — DISMISSED** [review]

**Review by rdenarie — DISMISSED** [review]

**Review by ahmadmido — APPROVED** [review]

**Pull request merged**